services/gateway/shared/moleculer-io/index.js

```diff
@@ -332,14 +333,24 @@ module.exports = {
 	},
 };
 
+function safeTestRegex(regex, action) {
+	try {
+		if (safeRegex(regex)) {
+			return regex.test(action);
+		}
+	} catch (e) {
+		return false;
+	}
+}
```

Comment:
If `safeRegex(regex)` returns false, it'll return `undefined`. Is that our expectation?
Also, can we please add some unit tests for `safeTestRegex` ?
```suggestion
	try {
		if (safeRegex(regex)) return regex.test(action);
	} finally {
		return false;
	}
}
```